spec/cordova/restore-util.spec.js

```diff
@@ -390,7 +356,7 @@ describe('cordova/restore-util', () => {
                 expectPluginsInPkgJson([{
                     name: 'cordova-plugin-camera',
                     spec: '^2.3.0',
-                    variables: { variable_1: 'value_1', variable_2: 'value_2', variable_3: 'value_3' }
+                    variables: { variable_1: 'value_1', variable_3: 'value_3' }
```

Comment:
Did you check whether the changed tests duplicate any of the old ones after the change?